.gitlab-ci.yml

```diff
@@ -0,0 +1,29 @@
+before_script:
+  - docker info
```

Comment:
Nice for your debugging, but is it useful in production?

Reply:
It's useful to have when a test fails, but as in our case we will always have access to the runners it's probably not needed. I'll remove it.